plugins/scanners/dos/src/main/kotlin/DosScanner.kt

```diff
@@ -200,6 +200,8 @@ class DosScanner internal constructor(
             return null
         }
 
+        logger.info { "Scan job added with ID '$id'." }
```

Comment:
Well actually it would be kind of helpful to have it in the issue itself too, because then you wouldn't necessarily need to dig into the logs to find it if it's needed.